src/examples/forms/range/RangeValidation.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Range, Message } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+type validationTypes = 'success' | 'warning' | 'error';
+
+const valType: Record<validationTypes, string> = {
```

Comment:
How would naming this variable `messages` or `validationMessages` sound?

I think it would be semantically more accurate because the value is a object the holds messages. Specifically, keys of validation types that point to message strings. Also "validation types" is referred to on lines 12 and line 14 feels like a semantic repeat.